projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.html

```diff
@@ -2,7 +2,7 @@
 <div *ngIf="node.question.renderingType === 'form'">
```

Comment:
Presumably in this file we should also translate the placeholders if possible right?